src/components/skills-quiz/data/service.js

```diff
@@ -24,3 +24,9 @@ export function postSkillsGoalsAndJobsUserSelected(goal, skillsId, interestedJob
     logError(new Error(error));
   });
 }
+
+export function fetchCourseEnrollments() {
+  const config = getConfig();
+  const url = `${config.LMS_BASE_URL}/api/enrollment/v1/enrollment`;
```

Comment:
The api will return enrollments for the request user, correct?

Reply:
yes, it will return the enrollments for the request user.